gc/verbose/VerboseHandlerOutput.cpp

```diff
@@ -102,6 +103,7 @@ MM_VerboseHandlerOutput::enableVerbose()
 {
 	/* Initialized */
 	(*_mmOmrHooks)->J9HookRegisterWithCallSite(_mmOmrHooks, J9HOOK_MM_OMR_INITIALIZED, verboseHandlerInitialized, OMR_GET_CALLSITE(), (void *)this);
+	(*_mmOmrHooks)->J9HookRegisterWithCallSite(_mmOmrHooks, J9HOOK_MM_OMR_REINITIALIZED, verboseHandlerReinitialized, OMR_GET_CALLSITE(), (void *)this);
```

Comment:
`J9HOOK_MM_OMR_REINITIALIZED` was added for CRIU. Why don't we wrap all code related to the hook with `J9VM_OPT_CRIU_SUPPORT` ifdefs?

Reply:
No use for it outside CRIU, wrapped it with `J9VM_OPT_CRIU_SUPPORT`